.github/ISSUE_TEMPLATE/request_maintainership.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Request Maintainership
+about: Use this template to apply to maintain a Laminas, Laminas API Tools, or Mezzio repository.
+title: "[MAINTAINER REQUEST]: [REPOSITORY]"
+labels: Maintainer Requseet
```

Comment:
```suggestion
labels: Maintainer Request
```

Reply:
Did not even had this PR on my radar. Nice 👍🏼